src/main/java/com/viaversion/viafabricplus/injection/mixin/features/movement/collision/MixinHoneyBlock.java

```diff
@@ -39,4 +53,49 @@ private static void simplifyVelocityComparisons(double d, CallbackInfoReturnable
         }
     }
 
+    @Inject(method = "onEntityCollision", at = @At("HEAD"), cancellable = true)
+    private void applyBedrockHoneyCollision(BlockState state, World world, BlockPos pos, Entity entity, EntityCollisionHandler handler, boolean bl, CallbackInfo ci) {
+        if (!ProtocolTranslator.getTargetVersion().equals(BedrockProtocolVersion.bedrockLatest)) {
+            return;
+        }
+        ci.cancel();
+        if (this.isSliding(pos, entity)) {
+            this.addCollisionEffects(world, entity);
+        }
+
+        final Vec3d velocity = entity.getVelocity();
```

Comment:
Is it supposed to modify the player velocity even when the player is not sliding?